http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/dao/HttpDockerDAOSpec.scala

```diff
@@ -160,8 +160,9 @@ class HttpDockerDAOSpec extends AnyFlatSpec with Matchers with BeforeAndAfterAll
         |}
         |""".stripMargin
     val expectedResult =
-      DecodingFailure("Attempt to decode value on failed cursor", List(DownField("config")))
-    decode[ContainerConfigResponse](jsonString) shouldBe Left(expectedResult)
+      DecodingFailure("Missing required field", List(DownField("config")))
+    decode[ContainerConfigResponse](jsonString).swap.toOption.get.getMessage shouldBe expectedResult.leftSideValue
```

Comment:
hmm.. why did this PR require updating the DockerDaoSpec?

Reply:
I think circe version gets updated transitively...so the error message is a bit different in newer version